### PR TITLE
Change pkg title + add NEWS.md

### PR DESCRIPTION
**Pull request by JohnMcIntyreJR**

### CODE REVIEW

**Review by csgillespie — DISMISSED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# jrNotes 0.2.2 _2020-09-22_
+  * Add NEWS.md
+  * Change pkg title
+  * Allowing 2 extra notes
```
Comment (csgillespie) [inline NEWS.md:4]: Can you add what notes were added.

**Review by csgillespie — CHANGES_REQUESTED** [review]

**Review by csgillespie — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,10 @@ language: R
 sudo: false
 cache: packages
 
+env:
+  global:
+    - ALLOWED_NOTES=3
+
```
Comment (csgillespie) [inline .travis.yml:8]: ALLOWED_NOTES nows appears twice in the travis.yml file

**Review by csgillespie — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (JohnMcIntyreJR) [reply .travis.yml]: Corrected

**Comment by JohnMcIntyreJR** [comment]
@csgillespie Still just allowing me to have the one note which is weird. Any idea why?

**Review by csgillespie — APPROVED** [review]

**Pull request merged**